ide/textmate.lexer/external/org.eclipse.tm4e.core-0.4.1-pack1-license.txt

```diff
@@ -0,0 +1,92 @@
+Name: TextMate support for Eclipse
+Version: 0.4.1-pack1
+License: EPL-v20
+Description: Used to interpret TextMate grammars
+Origin: https://github.com/eclipse/tm4e
+
+Eclipse Public License - v 2.0
```

Comment:
Possible mimatch: https://github.com/eclipse/tm4e/blob/master/LICENSE states EPL 1.0 even in the master version.

Reply:
Thank you for checking this. However I checked a few of the files in the core project, and I see EPL-v2.0 - checked:

https://github.com/eclipse/tm4e/blob/0.4.1/org.eclipse.tm4e.core/src/main/java/org/eclipse/tm4e/core/registry/Registry.java
https://github.com/eclipse/tm4e/blob/0.4.1/org.eclipse.tm4e.core/src/main/java/org/eclipse/tm4e/core/internal/grammar/Grammar.java
https://github.com/eclipse/tm4e/blob/0.4.1/org.eclipse.tm4e.core/src/main/java/org/eclipse/tm4e/core/internal/grammar/parser/Raw.java
https://github.com/eclipse/tm4e/blob/0.4.1/org.eclipse.tm4e.core/src/main/java/org/eclipse/tm4e/core/internal/grammar/reader/SyncGrammarReader.java
https://github.com/eclipse/tm4e/blob/0.4.1/org.eclipse.tm4e.core/src/main/java/org/eclipse/tm4e/core/internal/parser/xml/XMLPListParser.java
https://github.com/eclipse/tm4e/blob/0.4.1/org.eclipse.tm4e.core/src/main/java/org/eclipse/tm4e/core/grammar/IGrammar.java
https://github.com/eclipse/tm4e/blob/0.4.1/org.eclipse.tm4e.core/src/main/java/org/eclipse/tm4e/core/model/Tokenizer.java
https://github.com/eclipse/tm4e/blob/master/org.eclipse.tm4e.core/src/main/java/org/eclipse/tm4e/core/theme/RGB.java
https://github.com/eclipse/tm4e/blob/master/org.eclipse.tm4e.core/src/main/java/org/eclipse/tm4e/core/theme/css/CSSStyle.java
https://github.com/eclipse/tm4e/blob/0.4.1/org.eclipse.tm4e.core/src/main/java/org/eclipse/tm4e/core/model/DecodeMap.java